Makefile

```diff
@@ -714,7 +712,7 @@ bundle: bundle-generate bundle-crd-clean update-bundle bundle-validate bundle-im
 
 .PHONY: bundle-crd-clean
 bundle-crd-clean:
-	git checkout -- config/crd/bases/
+	git checkout -- config/crd/bases
```

Comment:
Is this change expected here?

Reply:
I took some changes from master, but reverted it.